src/typescript/07-functions/functions.md

```diff
@@ -113,21 +114,21 @@ dog.bark.call(address);
 
 Line 10 will error with: `Property 'name' is missing in type '{ street: string; }' but required in type '{ name: string; }'.`
 
-## Exercise: dnaCost
+## Exercise: DNA Cost
 
-### The Problem
+### The problem
```

Comment:
Please update these headers to match the structure here: https://bitovi.atlassian.net/wiki/spaces/BSG/pages/665976833/Bitovi+Academy+Style+Guide#Page-structure